azext_edge/edge/providers/support_bundle.py

```diff
@@ -86,6 +86,13 @@ def build_bundle(
         if ops_service in [OpsServiceType.auto.value, service_moniker]:
             deployed_apis = api_info["apis"].get_deployed()
 
+            if not deployed_apis:
+                expected_api_version = api_info["apis"].as_str()
+                logger.warning(
+                    f"Skip capturing CRD for {service_moniker} since {expected_api_version} is not deployed, "
```

Comment:
nit, suggesting something like:

"The following API(s) were not detected {expected_api_version}. CR capture for {service moniker} will be skipped. Still attempting capture of runtime resources...".

Reply:
Part of the nit is our system can support multiple apis per service.

Reply:
![image](https://github.com/user-attachments/assets/c7ce1d6e-b51d-49cf-9fe1-54b2412dd767)